apps/timetrace/ttanal

```diff
@@ -32,6 +33,46 @@ def tt_reader(paths: list[str]):
                     break
                 yield buf_to_rec(buf)
 
+time_units = [
+    ("ns", 0.000000001),
+    ("us", 0.000001),
+    ("ms", 0.001),
+    ("s", 1),
+    ("m", 60),
+    ("h", 3600),
+    ("d", 86400),
+    ("y", 31556952),
+    (None, None),
+]
+
+mem_units = [
+    ("B", 1),
+    ("kB", 1024),
+    ("MB", 1048576),
+    ("GB", 1073741824),
+    ("TB", 1099511627776),
```

Comment:
any chance of notating these numbers as `2**20, 2**30, 2**40` or at least sticking in underscores?